### ISSUE TITLE
Optimal hyperparameter selection for softmax  training

### ISSUE THREAD

**Issue by sankalpvarshney** [davidsandberg/facenet#1196]
Hi David,

Can you help me to guide how to choose optimal hypereparameter for this facenet model. I tried so many options all of the time i get under-fitting or over-fitting problem.
Please if it's possible provide me the best optical parameter for softmax
![Screenshot from 2021-03-01 13-34-43](https://user-images.githubusercontent.com/41926323/109468644-e6260f80-7a92-11eb-897e-47d7d18b39e5.png)
 training.

**Comment by vuhung142** [comment]
> Hi David,
> 
> Can you help me to guide how to choose optimal hypereparameter for this facenet model. I tried so many options all of the time i get under-fitting or over-fitting problem. Please if it's possible provide me the best optical parameter for softmax ![Screenshot from 2021-03-01 13-34-43](https://user-images.githubusercontent.com/41926323/109468644-e6260f80-7a92-11eb-897e-47d7d18b39e5.png) training.

@davidsandberg
